src/test/java/edu/ucsb/cs156/courses/controllers/UCSBAPIQuarterControllerTests.java

```diff
@@ -111,4 +111,40 @@ public void test_loadQuarters() throws Exception {
             response.getResponse().getContentAsString(),
             new TypeReference<List<UCSBAPIQuarter>>() {}));
   }
+
+  @Test
+  public void test_activeQuarters() throws Exception {
+
+    List<String> activeQuarterList = List.of("20244");
+    when(ucsbAPIQuarterService.getActiveQuarterList()).thenReturn(activeQuarterList);
+
+    String url = "/api/public/activeQuarters";
+    MvcResult response =
+        mockMvc
+            .perform(get(url).contentType("application/json"))
+            .andExpect(status().isOk())
+            .andReturn();
+
+    assertEquals(
+        activeQuarterList,
+        objectMapper.readValue(
+            response.getResponse().getContentAsString(), new TypeReference<List<String>>() {}));
+  }
+
+  //   @Test
+  //   public void test_getActiveQuarterList_S20_F19_1() throws Exception {
+  //     ArrayList<UCSBAPIQuarter> expected = new ArrayList<UCSBAPIQuarter>();
+  //     expected.add(new Quarter("S20"));
+  //     assertEquals(expected, Quarter.quarterList("20202", "20202"));
+  //   }
+
+  //   @Test
+  //   public void test_getActiveQuarterList_F19_S20() throws Exception {
+  //     List<Quarter> expected = new ArrayList<Quarter>();
+  //     expected.add(new Quarter("F19"));
+  //     expected.add(new Quarter("W20"));
+  //     expected.add(new Quarter("S20"));
+
+  //     assertEquals(expected, Quarter.quarterList("20194", "20202"));
+  //   }
```

Comment:
https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html#why-should-commented-out-code-be-removed